### PR TITLE
#306: Now the description field is not cropped in mobile devices

### LINKED ISSUE #306: Mobile Android > Keyboard hides the description textbox when user typing
# Actual Result

When the event description is longer more than 3 lines, the user cannot see the text he typing. The keyboard hides the description textbox. 

# Expected Result
When the keyboard show, it shouldn't hide the description textbox.

# Environment
DEV platform
Android Samsung A51, Chrome

### PR DESCRIPTION
**Pull request by huy-ta**
Resolves #306.

- Before:

Screenshot_20210312_152603_com android chrome

- After:

Screenshot_20210312_152354_com android chrome

### CODE REVIEW

File: webpack.commons.js [hunk 1]
```diff
@@ -71,6 +71,8 @@ module.exports = {
     })
   ],
   devServer: {
+    host: '0.0.0.0',
+    disableHostCheck: true,
```
Comment (rezk2ll) [inline webpack.commons.js:75]:
```suggestion
```

Comment (huy-ta) [reply webpack.commons.js:75]: Why? It was part of the PR.

Comment (rezk2ll) [reply webpack.commons.js:75]: maybe not everyone wants to expose their dev environment to the whole network by default.
other than that is should be in a separate PR.

Comment (huy-ta) [reply webpack.commons.js]: OK

**Review by rezk2ll — APPROVED** [review]

**Pull request merged**